### PR TITLE
Check out how much dead code I found

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

renaming variables and classes, I found that many of the most confusing classes weren't actually used!

### Changes proposed in this pull request

DELETE ALL THE DEAD CODE

### CODE REVIEW

**Comment by benilovj** [comment]
🔥

File: src/ManageCourses.Api/Controllers/DataController.cs [hunk 1]
```diff
@@ -40,22 +26,6 @@ public OrganisationCourses ExportByOrganisation(string ucasCode)
...
-        }
-
-        /// <summary>
-        /// Imports the reference data.
-        /// </summary>
-        [ApiTokenAuth]
-        [ExemptFromAcceptTerms]
-        [HttpPost]
-        [Route("referencedata")]
-        public void ImportReferenceData([FromBody] ReferenceDataPayload payload)
```
Comment (defong) [inline src/ManageCourses.Api/Controllers/DataController.cs]: You sure about this one?

Comment (the-daniel-rothig) [reply src/ManageCourses.Api/Controllers/DataController.cs]: Yes 🔥

**Comment by defong** [comment]
[remove all the dead code](https://trello.com/c/yEHuUgeS/524-remove-all-the-dead-code)

**Pull request merged**